### DISCUSSION TITLE
zh/guide/drivers/webdav

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#39] [Announcements]
# zh/guide/drivers/webdav

AList v3的文档

https://alist.nn.ci/zh/guide/drivers/webdav.html

**Comment by Magical-Lin** [comment]
如何让alist自身就可以把这些网盘挂载为本地路径呢？某些场景下的程序需要保存东西的目录只能填路径而不是网页链接，而这样我的主机就需要再去下载安装一个reclone来挂载alist为webdev本地目录。

**Reply by crarm** [reply]
对，这个需求很常见，感觉实现这个功能很复杂（或者不能实现？不太懂技术）
比如emby，只能填写本地路径或者共享路径，不能直接访问webdav，要使用网盘，还要挂载到本地再访问，使用rclone挂载到本地有很大的性能损失，在网页上下载满带宽，在挂载盘里复制只有3MB左右了。

**Reply by BrandonStudio** [reply]
你要找的是不是https://alist.nn.ci/zh/guide/webdav.html？
Windows下直接用WebClient就可以了呀

**Comment by zwh310306** [comment]
我的OD没有api，用alist 转换WebDAV链接 挂载会出现 PROPFIND /: 403 错误无法访问。

**Reply by sunyz1988** [reply]
把供应商改为SharePoint即可

**Reply by SkysCrystal** [reply]
同样的问题，换了 SharePoint 也是不行

**Reply by satellas** [reply]
老哥解决了吗，我的是教育版OD，同样的问题

**Reply by bohong65** [reply]
同样的问题，我也是教育版OD PROPFIND403

**Reply by YuanYao-git** [reply]
同样的问题，onedrive教育版a1权限

**Reply by SkysCrystal** [reply]
> 同样的问题，换了 SharePoint 也是不行

我过了段时间重新连接，又可以了不知道是不是跟管理员设置的权限有关

**Reply by ypq123456789** [reply]
老哥解决了吗，我也是同样的问题。是南大的世纪互联版

**Reply by 1hinhin** [reply]
同样的问题，教育版＋failed to list objs: PROPFIND /: 403

**Reply by Valeaf** [reply]
同样的问题，请问大家解决了吗

**Reply by friedrica137** [reply]
同，无api教育版onedrive，failed to list objs: PROPFIND ...: 404

**Reply by haocat** [reply]
我也 不过我发现是因为我们学校设置了密码过期的期限 登录onedrive网页就会提示强制修改密码 修改密码之后解决问题

**Reply by 75cnpl** [reply]
也遇到了这个问题，尝试了老哥说的网页推出重新登录之后还是没有解决

**Reply by hzj629206** [reply]
我是主动去修改密码后就没问题了。

**Reply by touse34** [reply]
还是遇到403的问题，请问有具体解决方法吗，我是onedrive教育版A1plus订阅

**Comment by bohong65** [comment]
关于无api的onedrive出现PROPFIND /: 403，请手动打开OD网页版，退出登录后重新登录，可能要求更新密码。我的教育版更新密码后恢复正常

**Reply by 75cnpl** [reply]
尝试了之后还是出现这个错误

**Comment by luobing2000** [comment]
添加本地WebDAV服务，显示“failed get objs: failed to list objs: Authorize http://*********:7080/: 401”，但是我用同样的账户信息在浏览器是可以正常访问此WebDAV服务的。

**Reply by heng9137** [reply]
你好，请问你的解决了吗，参数要怎么添加呢

**Reply by hubumulity** [reply]
海外账户都有两步验证，不行

**Comment by Super-Zoe** [comment]
![配置参考](https://user-images.githubusercontent.com/36558727/191271086-b4ecada6-2661-496c-a314-a47e13dd8013.png)

**Comment by onelxzy** [comment]
能不能去除限制webdav挂载onedrive的时候域名必须是com或者cn结尾啊。我想在访问国际版onedrive时使用worker代理，但是由于搭建在家里服务器，并无80/443端口，故无法使用alist提供的代理下载方式。想使用worker代理onedrive的webdav域名然后挂载经过代理的webdav直接得到走worker的链接。

**Comment by sglkmh** [comment]
使用alist通过webdav方式挂载另一个alist，里面的视频播放不了，但是用其他软件通过webdav方式挂载就可以播放视频，作者能否修复一下，谢谢了

**Comment by abiglee** [comment]
使用webdav挂世纪互联OneDrive速度只有1m/s左右，登录世纪互联OneDrive下载速度能达到40m/s多，
这正常么？

**Reply by abiglee** [reply]
我知道了，是我服务器带宽小。。。只有1m

**Reply by abiglee** [reply]
为什么这里webdav不能302重定向到OneDrive？

**Reply by JokerWFL** [reply]
方便教一下我怎么挂世纪互联吗

**Comment by xcz-ns** [comment]
无api的onedrive出现PROPFIND /: 403

**Comment by Chellane** [comment]
没看懂这个是怎么获取webdav的根地址的，请大佬们指导下

**Comment by lizhiwei8061** [comment]
挂载的webdav可以设置增加登录或者密码才能访问相关文件么

**Comment by hellofinch** [comment]
failed get objs: failed to list objs: Authorize http://**********/Documents/: 401
这是为什么呢？

**Comment by wzdc** [comment]
为什么WebDAV下载方式不能是302呀，这应该是可以的吧？

**Comment by WongKwongYu** [comment]
无权开api，也无权关authenticator，是不是就不能用alist了？？为啥rclone没这问题

**Reply by wdsjhskj** [reply]
> 无权开api，也无权关authenticator，是不是就不能用alist了？？为啥rclone没这问题

我觉得302重定向时可以使用 https://username:password@example.com 这种方式来传递auth，但是听网上的人说这种方式不安全 不过我觉得如果是私用的话应该没太大问题

**Comment by YFEGL** [comment]
seafile 的webdav报这个错failed get objs: failed to list objs: ReadDir，是不支持seafile吗？我直接在浏览器用是可以的

**Comment by xiykj** [comment]
你好，我用alist来挂载局域网中的web文件服务器时，是用的webdav方式，访问的时候报错 failed get objs: failed to list objs: PROPFIND /: 200 这个应该如何解决？

**Reply by yigedabuliu** [reply]
挂在webdav时，根路径应该是为/dav，例如/dav/[自己文件夹]而不是/[自己文件夹]

**Comment by jkjoy** [comment]
failed get objs: failed to list objs: Authorize https://jike.teracloud.jp/dav/: 401
